src/rpc/blockchain.h

```diff
@@ -53,6 +54,7 @@ void CalculatePercentilesByWeight(CAmount result[NUM_GETBLOCKSTATS_PERCENTILES],
  * @return a UniValue map containing metadata about the snapshot.
  */
 UniValue CreateUTXOSnapshot(
+    const bool is_compact,
```

Comment:
maybe `enum UTXOSnapshotFormat { COMPACT = 0, SQLITE = 1 };` would be nice and more explicit?

Reply:
Yes, I agree. I was going back and forth between them but I think explicit is better here. Would be easier to introduce another format in the future too. Will change 👍

Reply:
I see this is now addressed in 84a6e28a8536fdc79796856e36be86a3bffe1848, thanks!